core/src/main/java/com/googlecode/psiprobe/tools/AccessorFactory.java

```diff
@@ -19,11 +19,6 @@
 public class AccessorFactory {
 
   /**
-   * Instantiates a new accessor factory.
-   */
-  private AccessorFactory() {}
```

Comment:
Good catch!  My mistake.  Sonar likely flagged me.  Class needs to be final as well.  I'll fix that.  Thanks.  Appears there are a couple of these.  Sorry about that.  Sonars first complaint was not documenting them.  I thought they were all public.  Fix will come in next couple of days once you get a chance to look over the big pull request sitting out there :)